res/css/views/rooms/_RoomTile.scss

```diff
@@ -37,165 +38,138 @@ limitations under the License.
-                    margin-right: 7px;
-                }
+            &.mx_RoomTile_titleHasUnreadEvents {
+                font-weight: 600;
             }
+        }
 
-            // The context menu buttons are hidden by default
-            .mx_RoomTile_menuButton,
-            .mx_RoomTile_notificationsButton {
-                width: 20px;
-                min-width: 20px; // yay flex
-                height: 20px;
-                margin-top: auto;
-                margin-bottom: auto;
-                position: relative;
-                display: none;
+        .mx_RoomTile_subtitle {
+            line-height: $font-18px;
+            color: $secondary-content;
 
+            .mx_RoomTile_videoIndicator {
                 &::before {
-                    top: 2px;
-                    left: 2px;
+                    display: inline-block;
+                    vertical-align: text-bottom;
                     content: '';
+                    background-color: $secondary-content;
+                    mask-image: url('$(res)/img/element-icons/call/video-call.svg');
+                    mask-size: 16px;
                     width: 16px;
                     height: 16px;
-                    position: absolute;
-                    mask-position: center;
-                    mask-size: contain;
-                    mask-repeat: no-repeat;
-                    background: $primary-content;
+                    margin-right: 4px;
                 }
-            }
 
-            // If the room has an overriden notification setting then we always show the notifications menu button
-            .mx_RoomTile_notificationsButton.mx_RoomTile_notificationsButton_show {
-                display: block;
+                &.mx_RoomTile_videoIndicator_active {
+                    color: $accent;
+
+                    &::before {
+                        background-color: $accent;
+                    }
+                }
             }
 
-            .mx_RoomTile_menuButton::before {
-                mask-image: url('$(res)/img/element-icons/context-menu.svg');
+            .mx_RoomTile_videoParticipants::before {
+                display: inline-block;
+                vertical-align: text-bottom;
+                content: '';
+                background-color: $secondary-content;
+                mask-image: url('$(res)/img/element-icons/group-members.svg');
+                mask-size: 16px;
+                width: 16px;
+                height: 16px;
+                margin-right: 2px;
             }
         }
 
-        .mx_RoomTile_voiceChannel {
-            width: 100%;
-            display: flex;
-            align-items: center;
+        .mx_RoomTile_titleWithSubtitle {
+            margin-top: -3px; // shift the title up a bit more
+        }
+    }
 
-            .mx_FacePile {
-                margin: 6px 0 4px;
-            }
+    .mx_RoomTile_notificationsButton {
+        margin-left: 4px; // spacing between buttons
```

Comment:
I didn't actually touch this part of the CSS - the diff just makes it look like I did, since I removed a couple of wrapper classes that changed the indentation. Should I still update it?

Reply:
Oh right, just ignore then!